client/src/templates/Challenges/classic/mobile-layout.tsx

```diff
@@ -17,27 +16,26 @@ const mapStateToProps = createStructuredSelector({
   currentTab: currentTabSelector
 });
 
-const mapDispatchToProps = dispatch =>
+const mapDispatchToProps = (dispatch: Dispatch) =>
   bindActionCreators(
     {
       moveToTab
     },
     dispatch
   );
-
-const propTypes = {
-  currentTab: PropTypes.number,
-  editor: PropTypes.element,
-  guideUrl: PropTypes.string,
-  hasPreview: PropTypes.bool,
-  instructions: PropTypes.element,
-  moveToTab: PropTypes.func,
-  preview: PropTypes.element,
-  testOutput: PropTypes.element,
-  videoUrl: PropTypes.string
-};
-
-class MobileLayout extends Component {
+interface MobileLayoutProps {
+  currentTab: number;
+  editor: JSX.Element | null;
+  guideUrl: string;
+  hasPreview?: boolean;
+  instructions?: JSX.Element;
+  moveToTab: typeof moveToTab;
+  preview?: JSX.Element;
+  testOutput?: JSX.Element;
```

Comment:
```suggestion
  hasPreview: boolean;
  instructions: JSX.Element;
  moveToTab: typeof moveToTab;
  preview: JSX.Element;
  testOutput: JSX.Element;
```
Do these need to be optional?

Reply:
I don't think so @ojeytonwilliams

Reply:
Thanks @Ismailtlem